app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def flash_class(level)
```

Comment:
I would suggest to prepare method not only for class but for all flash message. Something like you have for `<a>` tag. In this case we use `link_to` method and this method generate whole HTML. The same you can do here with this flash message.